OrbitCore/FunctionUtils.h

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_CORE_FUNCTION_UTILS_H_
+#define ORBIT_CORE_FUNCTION_UTILS_H_
+
+#include <cstdint>
+#include <string>
+
+#include "OrbitFunction.h"
+#include "SamplingProfiler.h"
+#include "ScopeTimer.h"
+
+namespace function {
+
+inline const std::string& GetPrettyName(const Function& func) {
```

Comment:
probably GetDisplayName... not sure.

Reply:
Yes, the `GetPrettyName` is not very good name as the function doesn't simply return `pretty_name`. I'll rename it.

Reply:
just fyi In `symbol.proto` I switched away from `pretty_name` to 'demangled_name`